### DISCUSSION TITLE
Roadmap for later update of after make mac app؟ for main software lama cleaner

### DISCUSSION THREAD

**Discussion by wolfkingal2000** [Sanster/IOPaint#368] [Ideas]
Roadmap for later update of after make mac app؟ for main software lama cleaner went now new future or enhancement?

**Comment by Sanster** [comment]
Here are some features I plan to implement in the next version:
- Add sdxl-inpainting model: https://huggingface.co/diffusers/stable-diffusion-xl-1.0-inpainting-0.1
- Add kandinsky-2.2-inpainting model: https://huggingface.co/kandinsky-community/kandinsky-2-2-decoder-inpaint
- MobileSAM: https://github.com/Sanster/lama-cleaner/issues/352
- A simple cli tool to do bulk image inpainting (only LaMa & MAT model)
- Out-painting

**Reply by xingren23** [reply]
maybe no need to follow up huggingface's sdxl-inpainting model.

https://github.com/lllyasviel/Fooocus/discussions/414 
Using Diffusers’ XL inpaint model. This will cause less distortions and you can get useable results in a few attempts. But this Diffusers model is not trained by StabilityAI’s dataset, which means using this model will influence the style of base models. This also influences fine-tuned models/loras/offset-merges. Models will look like they are train on diffusers dataset (probably LAION style).

**Comment by wolfkingal2000** [comment]
> Here are some features I plan to implement in the next version:
> 
> * Add sdxl-inpainting model: https://huggingface.co/diffusers/stable-diffusion-xl-1.0-inpainting-0.1
> * Add kandinsky-2.2-inpainting model: https://huggingface.co/kandinsky-community/kandinsky-2-2-decoder-inpaint
> * MobileSAM: [[Feature Request] Add MobileSAM model in interactive segment plugin #352](https://github.com/Sanster/lama-cleaner/issues/352)
> * A simple cli tool to do bulk image inpainting (only LaMa & MAT model)
> * Out-painting


Thanks for the answer, if you can add more settings for better control on inpainting like denoise strength and add lora model and add vae model and add more control net  model and can better model manger can download inside app for custom model were  good
and look this comment i think add this can better inpainting 
https://github.com/Sanster/lama-cleaner/issues/359#issuecomment-1701329955

**Comment by Kingal20000** [comment]
> Here are some features I plan to implement in the next version:
> - Add sdxl-inpainting model: https://huggingface.co/diffusers/stable-diffusion-xl-1.0-inpainting-0.1
> - Add kandinsky-2.2-inpainting model: https://huggingface.co/kandinsky-community/kandinsky-2-2-decoder-inpaint
> - MobileSAM: https://github.com/Sanster/lama-cleaner/issues/352
> - A simple cli tool to do bulk image inpainting (only LaMa & MAT model)
> - Out-painting

Hi can you make a project  panel to show what part off developing lama cleaner? Please and you tinking when upcoming new version?

**Comment by Sanster** [comment]
Hi, thank you for your continued interest in this project. I am currently working on version 2.0, The overall design goal is to make lama-cleaner a simple and easy-to-use inpainting/outpainting tool。The new version of the backend code has almost been completed, with the main remaining workload in the frontend. In order to easily add other features in the future, I am currently refactoring the frontend code. While working full-time, I may only have an average of 1 hour per day to work on development, I am not sure when it will be completed, but I hope to finish it as soon as possible.
